headers.js

```diff
@@ -8,6 +8,12 @@ const cspMeta = Object.entries({
     'https://gas.api.0x.org',
     'https://api.coingecko.com',
     'https://api.yearn.finance',
+    'https://test-api.yearn.network/v1/chains/1/vaults/getDynamic',
+    'https://test-api.yearn.network/v1/chains/1/strategies/metadata/get',
+    'https://test-api.yearn.network/v1/chains/1/vaults/get',
+    'https://test-api.yearn.network/v1/chains/1/vaults/earnings/get',
+    'https://api.thegraph.com/subgraphs/name/salazarguille/yearn-vaults-v2-subgraph-mainnet',
+    'https://raw.githubusercontent.com/yearn/yearn-assets/master/icons/aliases.json',
```

Comment:
```suggestion
    'https://test-api.yearn.network/v1/',
    'https://api.thegraph.com/subgraphs/name/salazarguille/yearn-vaults-v2-subgraph-mainnet/',
    'https://raw.githubusercontent.com/yearn/yearn-assets/',
```
I think this slightly-simplified version would make sense, though I do wonder if `test-api.yearn.network` is the right endpoint. Will this change in prod, and where does the URL come from? If it's configurable, it might be better to pull it in from `.env` or whatever.

Note that if a path is included in a CSP directive, a trailing slash means it's a prefix, whereas no trailing slash requires an exact match. I've added a trailing slash to the `api.thegraph.com` as sort of a guess here, since loading that page redirects to `https://api.thegraph.com/subgraphs/name/salazarguille/yearn-vaults-v2-subgraph-mainnet/graphql` in my browser, but double-check me on that.

Reply:
Great questions @reidrankin. For the question of yearn.network. All of these urls came from network calls when using the yearn-sdk. The yearn.network url is hardcoded in their sdk here https://github.com/yearn/yearn-sdk/blob/master/src/context.ts on line 66. From what I can tell, that is from the official yearn github account that is linked to from yearn.finance.

For the question about the subgraph url, that one also came from their sdk here: https://github.com/yearn/yearn-sdk/blob/master/src/services/subgraph/index.ts on line 16. The official yearn github account forked the salazarguille repo here https://github.com/yearn/yearn-vaults-v2-subgraph.

For the githubusercontet url, that was just for asset icons.

As for the data itself. The data from these endpoints used by our app is for calculating and displaying pricing data for yearn assets. So though our app doesn't source _balances_ for yearn assets from these, it does use the balance from unchained in combination with this data to calculate the price in USD displayed and the price history charts. So, assuming maliciousness from these data sources, the effect to the user could be that they see a different price value for their balance on yearn assets, though their underlying asset balance would still be from unchained.

Reply:
For whatever reason, the trailing slash on the subgraph url does break the call to get historical pricing data. So I removed it.

Reply:
Verified -- and sorry for the delay!

(Looking at the Yearn SDK more generally, they _really_ like hardcoding URLs to random crap, so I doubt there's a scalable solution here...)